php/libraries/LorisForm.class.inc

```diff
@@ -838,11 +843,17 @@ class LorisForm
      */
     function hiddenHTML($el, $type='hidden')
     {
-        $value = '';
-        if (isset($el['value'])) {
-            $value = "value=\"$el[value]\"";
+        $attributes = '';
+        foreach ($el as $key=>$value) {
+            if ($key=='type') {
+                $value = $type;
+            }
+            if (isset($value)) {
+                $attributes .= " $key=\"$value\"";
+            }
+
         }
-        return "<input name=\"$el[name]\" type=\"$type\" $value>";
+        return "<input $attributes>";
```

Comment:
@jstirling91 I had the option to just append some code to add the additional attributes seperately but it seemed better this way. all attributes are added in the same for loop and the type is forced to hidden (as it was previously)